### DISCUSSION TITLE
考试模式

### DISCUSSION THREAD

**Discussion by arcticovo** [ClassIsland/ClassIsland#560] [插件功能请求]
### 检查清单

- [X] 我已将本应用更新到最新测试版 ![最新的版本号](https://img.shields.io/github/v/release/ClassIsland/ClassIsland?include_prereleases&style=flat-square&label=)，并看过[最新提交](https://github.com/ClassIsland/ClassIsland/commits/dev/)，确认我想要的功能还没有实现。
- [X] 我已在 [Issues](https://github.com/ClassIsland/ClassIsland/issues?q=label%3A新功能) 和 [ClassIsland 文档](https://docs.classisland.tech/zh-cn/latest/app/)中检索，确认我想要的功能未被提交过。
- [X] 我认为我想要的功能是用户广泛需要的，插件不能替代的，且没有超出 ClassIsland 作为**灵动课表**的开发目标，而非添加与课表及教学辅助无关的内容。（您可以学习[编写插件](https://docs.classisland.tech/zh-cn/latest/dev/plugins/create-project/)来实现特殊的需求）。
- [X] 我已知晓由于开发者时间精力有限，加上已经积压了大量的功能请求，我提交的功能请求可能需要比较久的时间才能实现。

### 背景与动机

大屏经常被当wifi用，有些时候考试时忘记关会发出声音提醒影响考试。

### 想要实现或优化的功能

考试模式在考试前一键关闭提醒功能，**显示考试科目，时间，考试安排情况**

### 最后一步

- [X] 我认为上述的描述已经足以详细，以便开发人员理解您想要的功能。如果我的 issue 没有按照上述的要求填写，可能会被无条件关闭。

**Comment by MKStoler4096** [comment]
> 考试模式在考试前一键关闭提醒功能，**显示考试科目，时间，考试安排情况**

后三项推荐你用DSZ考试看板（）

**Comment by DryIce-cc** [comment]
此功能是 #320 的一部分。
ClassIsland 主程序大概不会实现此功能，与电视猪考试看板联动，或使用插件是不错的方法。

**Comment by baoliaojiang** [comment]
似乎dsz考试展板并没有语音提醒功能，而在#444有具体写语音提醒

**Comment by MKStoler4096** [comment]
> 似乎dsz考试展板并没有语音提醒功能，而在#444有具体写语音提醒

你来我这 https://github.com/ProjectCampus-CH/dsz-exam-showboard-next 提个issue（）

我研究一下edge tts（） @baoliaojiang

**Comment by arcticovo** [comment]
（更多）增加专注模式（类似_Windows专注模式_）

1. 开启时屏蔽提醒，关闭声音
2. 可设置持续时间，日程计划。
3. 和dsz联动的考试看板功能。
